.env.jenkins

```diff
@@ -3,4 +3,4 @@ WALLET_WIP_ENABLED=1
 NOTIFICATIONS_WIP_ENABLED=1
 DEBUG_LOGS_ENABLED=1
 STUB_STATUS_GO=0
-
+NETWORK_SWITCHING=1
```

Comment:
Can we add explicit NETWORK_SWITCHING=0 to prod?

Reply:
ok